torch/_dynamo/output_graph.py

```diff
@@ -686,14 +684,6 @@ def wrap_name(module_key):
 
         elif isinstance(target, torch.nn.Module):
             assert isinstance(target, torch.nn.Module)
-            if nnmodule_has_hooks(
```

Comment:
There's an `assert "source" in options` above, meaning this is always a known object, meaning it gets covered under the https://github.com/pytorch/pytorch/pull/108903 PR